src/constants.ts

```diff
@@ -124,7 +124,6 @@ const Constants = {
         uploadInterval: 0, // Maximum milliseconds in between batch uploads, below 500 will mean immediate upload
     },
     DefaultUrls: {
-        v1SecureServiceUrl: 'jssdks.mparticle.com/v1/JS/',
         v2SecureServiceUrl: 'jssdks.mparticle.com/v2/JS/',
```

Comment:
Shouldn't we also remove v2?

Reply:
I initially _did_ v2 also, but v2 is actually used when creating the /forwarder url when we report forwarding stats, so we need to keep this.